docs/docs/reference/contextual/derivation.md

```diff
@@ -371,6 +371,16 @@ ConstrApps        ::=  ConstrApp {‘with’ ConstrApp}
                     |  ConstrApp {‘,’ ConstrApp}
 ```
 
+Note: To align `extends` clauses and `derives` clauses, Scala 3 also allows multiple
+extended types to be separated by commas. So the following is now legal:
+```
+class A extends B, C { ... }
+```
+It is equivalent to the old form
+```
+class A extends B with C { ... }
+```
```

Comment:
Will the `with` form be deprecated? Should we say something about which form is recommended?